libevm/lib/evm_test.go

```diff
@@ -125,13 +177,27 @@ func TestEvmCallTracerWithTracerConfig(t *testing.T) {
 			DisableStack:     false,
 			DisableStorage:   false,
 			EnableReturnData: true,
```

Comment:
These settings don't do anything when it is not the struct tracer, right? If so, might omit them.

Reply:
Yes, they are ignored by the geth implementation in case the tracer is not the default one, removed from the unit test.